src/poller.ts

```diff
@@ -55,7 +55,7 @@ export class Poller {
           break;
         // ETH
         case 'Hccuen6RkUgEvyL9oSXW8ai9QiQaAiL8ESaqjp9oymBf':
-        case '7vfCXTUXx5WJV5JADk17DUJ4ksgau7utNKj4b963voxs':
+        case '2FPyTwcZLUg1MDrwsyoP4D6s1tM7hAkHYRjkNb5w6Pxk':
           decimals = 7;
```

Comment:
6

Reply:
shouldn't this have been 8 before for weETH?

Reply:
probably. we never test with eth, so wouldnt have noticed